### DISCUSSION TITLE
Touchpad slider not working Asus x550v

### DISCUSSION THREAD

**Discussion by xValek** [sebanc/brunch#805] [Q&A]
Hi to everyone,i istalled chromeos v83 and my touchpad slider is not working,then updated to v87 and nothing changed,left click work and 2 fingers for right click work also,only slides not work,any solution?Thank u

**Comment by Warrentheo** [comment]
The "Touchpad Slider" is not actually a real part of the hardware, it just is a faked feature added by the windows drivers for that hardware...  In other words, it is a software program running in the windows kernel...  So anything that is not running those windows drivers wont have it...
